lib/assets/sass/prototype.scss

```diff
@@ -5,11 +5,7 @@
 @import ".tmp/sass/plugins";
 
 // Patterns that aren't in Frontend
```

Comment:
can we update the comment to talk about supporting pages made with old templates, something like that?

Reply:
Fixed as suggested